06-building-a-program-start-to-finish/writing-tests/Output/test/spec/inverted-index.spec.js

```diff
@@ -0,0 +1,153 @@
+const chai = require('chai');
+const InvertedIndex = require('../../src/js/inverted-index.js');
+const books = require('../books.json');
+const books2 = require('../books2.json');
+const emptyBook = require('../emptyBook.json');
+const invalidBook = require('../invalidBook.json');
+
+const expect = chai.expect;
+const newIndex = new InvertedIndex();
+const sampleData = ' Mary had a    little #$%^6 lamb, a little lamb Mary had';
+
+
+describe('Validate files', () => {
+  it('should check that uploaded file is valid JSON', () => {
+    expect(newIndex.isValidFile(books)).to.be.true;
+    expect(newIndex.isValidFile(emptyBook)).to.be.false;
+    expect(newIndex.isValidFile(invalidBook)).to.be.false;
+  });
+
+  it('should not be empty', () => {
```

Comment:
This doesn't read as nicely as the spec above.

`Validate files ... should not be empty`.